gateways/istio-egress/templates/autoscale.yaml

```diff
@@ -10,7 +10,15 @@ metadata:
     release: {{ .Release.Name }}
 spec:
   maxReplicas: {{ $gateway.autoscaleMax }}
+{{- if $gateway.autoscaleMin }}
   minReplicas: {{ $gateway.autoscaleMin }}
```

Comment:
Seem still an issue here, what if `autoscaleMin` is 1 and the actual replica is 1, in that case, the default PDB(with minAvailable=1 ) still prevents node drain.